consutil/main.py

```diff
@@ -28,7 +29,7 @@ def show():
     devices = getAllDevices()
     busyDevices = getBusyDevices()
 
-    header = ["Line", "Baud", "PID", "Start Time"]
+    header = ["Line", "Actual/Configured Baud", "PID", "Start Time"]
```

Comment:
what else can this baud rate be? what is the reason to add it?

Reply:
can they be different?

Reply:
Actual baud is the baud rate that would actually be used to connect to the device, and configured baud is the baud rate that is pulled from config db. If the baud rate is successfully pulled from config db then the two are the same, but if a baud rate can't be pulled from config db, this is indicated by actual baud being "9600" (i.e. default value) and configured baud being "-"